x-pack/plugins/security/server/authentication/can_redirect_request.ts

```diff
@@ -11,6 +11,14 @@ const ROUTE_TAG_API = 'api';
 const KIBANA_XSRF_HEADER = 'kbn-xsrf';
 const KIBANA_VERSION_HEADER = 'kbn-version';
 
+export const API_ROUTES_SUPPORTING_REDIRECTS = [
+  '/api/security/logout',
+  '/api/security/saml/callback',
+  '/api/security/oidc/callback',
+  '/api/security/oidc/initiate_login',
+  '/api/security/v1/oidc',
```

Comment:
There are also some reporting endpoints that likely support redirects:

1) `/api/reporting/jobs/download/${jobId}` can be used to retrieve a previously generated report. Not sure if this is used in an XHR context at all, or if it's only ever used as a top-level request. Perhaps @elastic/kibana-reporting-services can help verify this?
2) `/api/reporting/generate/${reportType}?...` can be used to generate a report on the fly. There is a deprecated `sync` parameter you can pass in here, but it looks like it's being removed so perhaps it's not worth catering to this endpoint.

Reply:
I can add these, but do we officially recommend "navigating" to these URLs directly via browser? Those mentioned above are in our official docs (except for `/logout`).

Reply:
The reporting management interface has direct links to the `/api/reporting/jobs/download/${jobId}` route. Clicking the "download" icon in the grid opens a new tab to this endpoint, where the report is rendered.

Reply:
Ahhhh, I should have checked that, thanks! Then, it certainly deserves to be in this "allows redirects" list.